lib/chromic_pdf/api.ex

```diff
@@ -60,6 +61,32 @@ defmodule ChromicPDF.API do
     |> Keyword.put_new(source_type, source)
   end
 
+  defp rewrite_wait_for(opts) do
+    opts
+    |> Keyword.pop(:wait_for)
+    |> do_rewrite_wait_for()
```

Comment:
I came to the habit on piping these into a `case`, so you do not need the `do_` version.

Reply:
:) 👍  makes sense, but I kind of like the `do_` version here as otherwise I have so much indentation